Analytics/Request/BlockingRequestHandler.cs

```diff
@@ -225,19 +225,28 @@ public async Task MakeRequest(Batch batch)
                     if (_client.Config.CompressRequest)
                         content.Headers.ContentEncoding.Add("gzip");
 
-                    var response = await _httpClient.PostAsync(uri, content).ConfigureAwait(false);
+                    HttpResponseMessage response = null;
+                    bool retry = false;
+                    try
+                    {
+                        response = await _httpClient.PostAsync(uri, content).ConfigureAwait(false);
+                    }
+                    catch (System.Threading.Tasks.TaskCanceledException)
```

Comment:
I can't find documentation that suggests TaskCanceledException to be thrown here. Instead, it looks like we should catch HttpRequestException instead (https://stackoverflow.com/questions/40677708/which-exceptions-can-httpclient-throw). Can you verify?